tests/python/unit/test_binary_model.py

```diff
@@ -0,0 +1,44 @@
+import sys
+sys.path.append("..")
```

Comment:
No fiddling with the PATH, especially with relative imports. It is a dangerous habit. If you need to do it, then do it in the makefile.

Reply:
@david-behar